src/GraphQL/Types/Composite/InputObjectGraphType.cs

```diff
@@ -15,6 +21,19 @@ public class InputObjectGraphType : InputObjectGraphType<object>
     /// <inheritdoc cref="IInputObjectGraphType"/>
     public class InputObjectGraphType<TSourceType> : ComplexGraphType<TSourceType>, IInputObjectGraphType
     {
+        /// <inheritdoc/>
```

Comment:
```suggestion
        /// <summary>
        /// Converts a supplied dictionary of keys and values to an object.
        /// The default implementation uses <see cref="ObjectExtensions.ToObject"/> to convert the field values
        /// of the input object into an object of type <typeparamref name="TSourceType"/>.
        /// Overriding this method allows for customizing the deserialization process of input objects,
        /// much like a field resolver for output objects.
        /// </summary>
```
Uhh...how about this.???

Reply:
👍